README.md

```diff
@@ -2,7 +2,7 @@ codecentric's Spring Boot Admin
 ===============================
 [![Apache License 2](https://img.shields.io/badge/license-ASF2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0.txt)
 [![Build Status](https://github.com/codecentric/spring-boot-admin/workflows/build/badge.svg)](https://github.com/codecentric/spring-boot-admin/actions)
-[![Coverage Status](https://coveralls.io/repos/github/codecentric/spring-boot-admin/badge.svg?branch=master)](https://coveralls.io/github/codecentric/spring-boot-admin?branch=master)
+[![codecov](https://codecov.io/gh/codecentric/spring-boot-admin/branch/master/graph/badge.svg?token=u5SWsZpj5S)](https://codecov.io/gh/codecentric/spring-boot-admin)
```

Comment:
Is this correct? it seems we are always using the master branch. Yup, seems so

<img width="160" alt="image" src="https://user-images.githubusercontent.com/59687842/107158197-aafb5980-6988-11eb-818c-a9800bd23205.png">